services/vehicle-assembly-process-defect-detection-model-service/app/routers/predict.py

```
@@ -0,0 +1,240 @@
+
+    except (InvalidImageException, ImageSizeException, UnsupportedImageFormatException) as e:
+        logger.warning(f"이미지 검증 실패 - Request ID: {request_id}, 오류: {str(e)}")
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST,
+            detail={
+                "success": False,
+                "message": str(e),
+                "request_id": request_id,
+                "timestamp": datetime.now().isoformat()
+            }
+        )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)